Sources/Card/Models/Card.swift

```diff
@@ -0,0 +1,25 @@
+import Foundation
+
+public struct Card: Codable {
+
+    enum CodingKeys: String, CodingKey {
+        case number
+        case expirationDate = "expiry"
+        case securityCode = "security_code"
```

Comment:
Do we need to do all of this? Can we just do `JSONDecoder.KeyDecodingStrategy.convertFromSnakeCase`? I think we can assume things will be snake case from `orders/v2` for now. If that changes, we can look at workarounds if needed.

This would simplify our enum to then just be:

```
struct Card: Codable {
    case number
    case expirationDate
    case securityCode
}
```

It would also simplify things for PaymentsCore.

Reply:
Steven & I updated to simplify this.

Reply:
thank you :smile: !!!!